TestInterfaceClose fails on OSX

```
--- FAIL: TestInterfaceClose (0.00s)
	interface_test.go:15: *net.OpError route ip+net: no such network interface
FAIL
...
```
On: OSX Yosemite + El Capitan
`go version go1.6.2 darwin/amd64`

The interface is called `lo0` instead but it would still fail with ` *os.PathError open /dev/bpf0: permission denied` on my boxes.

What are the supported OS? I can provide a patch to deactivate the test via build tag for `darwin` if it makes sense.

--- a/block/aoe/interface_test.go
+++ b/block/aoe/interface_test.go
@@ -1,13 +1,14 @@
 package aoe
 
 import (
+	"net"
 	"sync"
 	"syscall"
 	"testing"
 )
 
 func TestInterfaceClose(t *testing.T) {
-	ai, err := NewInterface("lo")
+	ai, err := NewInterface(findLoopbackInterface(t))
 	if err != nil {
 		if err == syscall.EPERM {
 			t.Skipf("test must be run as root: %v", err)
@@ -35,4 +36,15 @@ func TestInterfaceClose(t *testing.T) {
 	}
 
 	wg.Wait()
+}
+
+func findLoopbackInterface(t *testing.T) string {
+	for _, name := range []string{"lo", "lo0"} {
+		if _, err := net.InterfaceByName(name); err == nil {
+			return name
+		}
+	}
+
+	t.Skip("could not find suitable loopback interface for test")
+	return ""
 }

